diff -Naur automake-1.16.1-orig/Makefile.in automake-1.16.1/Makefile.in
--- automake-1.16.1-orig/Makefile.in	2018-07-04 02:53:33.213186934 +0300
+++ automake-1.16.1/Makefile.in	2018-03-12 00:21:08.000000000 +0300
@@ -3367,7 +3367,7 @@
 check-am: all-am
 	$(MAKE) $(AM_MAKEFLAGS) check-TESTS check-local
 check: check-am
-all-am: Makefile $(INFO_DEPS) $(SCRIPTS) $(MANS) $(DATA)
+all-am: Makefile $(INFO_DEPS) $(SCRIPTS) $(DATA)
 installdirs:
 	for dir in "$(DESTDIR)$(bindir)" "$(DESTDIR)$(infodir)" "$(DESTDIR)$(man1dir)" "$(DESTDIR)$(amdir)" "$(DESTDIR)$(automake_acdir)" "$(DESTDIR)$(automake_internal_acdir)" "$(DESTDIR)$(docdir)" "$(DESTDIR)$(perllibdir)" "$(DESTDIR)$(pkgvdatadir)" "$(DESTDIR)$(scriptdir)" "$(DESTDIR)$(system_acdir)" "$(DESTDIR)$(perllibdir)"; do \
 	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
@@ -3434,7 +3434,7 @@
 	install-dist_automake_internal_acDATA install-dist_docDATA \
 	install-dist_perllibDATA install-dist_pkgvdataDATA \
 	install-dist_scriptDATA install-dist_system_acDATA \
-	install-info-am install-man install-nodist_perllibDATA
+	install-info-am install-nodist_perllibDATA
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-am
