diff -Naur ccache-3.4.2-orig/src/ccache.c ccache-3.4.2/src/ccache.c
--- ccache-3.4.2-orig/src/ccache.c	2018-08-16 22:51:32.886536436 +0300
+++ ccache-3.4.2/src/ccache.c	2018-03-25 23:24:05.000000000 +0300
@@ -2510,6 +2510,21 @@
 			free(relpath);
 			continue;
 		}
+		// Alternate form of specifying sysroot without =
+		if (str_eq(argv[i], "--sysroot")) {
+			if (i == argc-1) {
+				cc_log("Missing argument to %s", argv[i]);
+				stats_update(STATS_ARGS);
+				result = false;
+				goto out;
+			}
+			args_add(stripped_args, argv[i]);
+			char *relpath = make_relative_path(x_strdup(argv[i+1]));
+			args_add(stripped_args, relpath);
+			i++;
+			free(relpath);
+			continue;
+		}
 		if (str_startswith(argv[i], "-Wp,")) {
 			if (str_eq(argv[i], "-Wp,-P")
 			    || strstr(argv[i], ",-P,")
